clients/fides-js/src/components/notices/NoticeOverlay.tsx

```diff
@@ -263,37 +266,43 @@ const NoticeOverlay: FunctionComponent<OverlayProps> = ({
         onClose,
         onSave,
         onManagePreferencesClick,
-      }) => (
-        <ConsentBanner
-          bannerIsOpen={isOpen}
-          dismissable={isDismissable}
-          onOpen={dispatchOpenBannerEvent}
-          onClose={() => {
-            onClose();
-            handleDismiss();
-          }}
-          i18n={i18n}
-          isEmbedded={isEmbedded}
-          renderButtonGroup={({ isMobile }) => (
-            <NoticeConsentButtons
-              experience={experience}
-              i18n={i18n}
-              onManagePreferencesClick={onManagePreferencesClick}
-              enabledKeys={draftEnabledNoticeKeys}
-              onSave={(
-                consentMethod: ConsentMethod,
-                keys: Array<PrivacyNotice["notice_key"]>
-              ) => {
-                handleUpdatePreferences(consentMethod, keys);
-                onSave();
-              }}
-              isAcknowledge={isAllNoticeOnly}
-              isMobile={isMobile}
-              options={options}
-            />
-          )}
-        />
-      )}
+      }) => {
+        const isAcknowledge =
+          isAllNoticeOnly ||
+          experience.experience_config?.layer1_button_options ===
+            Layer1ButtonOption.ACKNOWLEDGE;
+        return (
+          <ConsentBanner
+            bannerIsOpen={isOpen}
+            dismissable={isDismissable}
+            onOpen={dispatchOpenBannerEvent}
+            onClose={() => {
+              onClose();
+              handleDismiss();
+            }}
+            i18n={i18n}
+            isEmbedded={isEmbedded}
+            renderButtonGroup={() => (
+              <NoticeConsentButtons
+                experience={experience}
+                i18n={i18n}
+                onManagePreferencesClick={onManagePreferencesClick}
+                enabledKeys={draftEnabledNoticeKeys}
+                onSave={(
+                  consentMethod: ConsentMethod,
+                  keys: Array<PrivacyNotice["notice_key"]>
+                ) => {
+                  handleUpdatePreferences(consentMethod, keys);
+                  onSave();
+                }}
+                isAcknowledge={isAcknowledge}
+                hideOptInOut={isAcknowledge}
```

Comment:
do we also need to hide opt in/out when `isSaveOnly==true` here?

Reply:
I didn't change the "Save Only" scenario at all. It was only a factor for modal before, so I kept it that way here.